### ISSUE TITLE
Phasing as Phantom causes suffocation damage

### ISSUE THREAD

**Issue by ArcusMG** [EdwinMindcraft/origins-architectury#273]
### Version Information

origins-forge-1.19.2-1.5.0.6-all
caelus-forge-1.19.2-3.0.0.6

### Expected Behavior

Expected to to be able to move into solid blocks with the phantom origin, without taking damage (though you should lose some hunger).

### Actual Behavior

when moving into a solid block, started taking suffocation damage.

### Reproduction Steps

1. load these mods:
canary
forge loader
origins for forge
and origin's library caelus
2. select the phantom origin when loading a world for the first time
3. attempt to move into a solid block
4. start suffocating

### Other Information

Not sure if this is an issue on the Origins mod end or Canary. I've reported it to both. Other mods installed:
forge 43.1.47
canary-mc1.19.2-0.0.7

**Comment by ArcusMG** [comment]
A link to the bug report for Canary:
https://github.com/AbdElAziz333/Canary/issues/50

**Comment by EdwinMindcraft** [comment]
The issue is known, both on forge and fabric with every single fork of lithium.
Lithium effectively removes a "method" (it's a lambda call) that origins uses.
The method stated by the Canary dev should work (it worked with both Lithium and Radium).

As far as I'm aware, you can't use a mixin to target another mixin, and coremods can't target methods with dynamic names.
TL;DR: I don't know of a method to fix this, use the workaround, the performance impact should be minimal for 90% of uses.
